smart_ats/jobs/api/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from taggit_serializer.serializers import TaggitSerializer, TagListSerializerField
+
+from smart_ats.jobs.models import Job
+
+
+class JobSerializer(TaggitSerializer, serializers.ModelSerializer):
```

Comment:
separate creation from retrieval.
the company, category, and author should consist of objects not string in retrieval, and on creation should consist of ids.